src/html/index.html

```diff
@@ -184,6 +184,20 @@ <h2>Button Actions</h2>
 @@if not isTX:
 			<div class="mui-tabs__pane" id="pane-justified-3">
 				<div class="mui-panel">
+					<div id="model_tab">
```

Comment:
```suggestion
					<div id="mixing_tab">
```

Reply:
Using the same id for this and the PWM model tab breaks hiding for non-PWM and disallows mixing for non-PWM; only the first 'model_tab' is hidden, so channel mixer tab is hidden for non-PWM (which it should not), and the PWM tab is shown but has no text fields.

Reply:
Thanks for fixing that!